main/lsp/LSPTask.cc

```diff
@@ -223,7 +224,10 @@ LSPTask::filterAndDedup(const core::GlobalState &gs,
               [](const unique_ptr<core::lsp::QueryResponse> &a, const unique_ptr<core::lsp::QueryResponse> &b) -> bool {
                   auto aLoc = a->getLoc();
                   auto bLoc = b->getLoc();
-                  auto cmp = (aLoc.beginPos() - bLoc.beginPos());
+                  int cmp = aLoc.file().id() - bLoc.file().id();
```

Comment:
This logic was not sorting properly before because `auto` inferred an unsigned type. 🤦